prime-router/quick-test.sh

```diff
@@ -22,8 +22,11 @@ NC='\033[0m' # No Color
 # let's make it possible for run for different states
 RUN_AZ=0
 RUN_FL=0
-RUN_ND=0
 RUN_LA=0
+RUN_ND=0
+RUN_NM=0
+RUN_TX=0
+RUN_VT=0
 # always should run, but we'll leave this here for now in case that could change at some point
```

Comment:
For consistency's sake, I moved around the options so they're in alphabetic order